components/vision.py

```diff
@@ -51,45 +52,44 @@ def __init__(
         self.camera_to_robot = self.robot_to_camera.inverse()
         self.last_timestamp = -1
         self.last_recieved_timestep = -1.0
-
-        self.single_best_log = field.getObject(name + "single_best_log")
-        self.single_alt_log = field.getObject(name + "single_alt_log")
-        self.multi_best_log = field.getObject(name + "multi_best_log")
-        self.multi_alt_log = field.getObject(name + "multi_alt_log")
-        self.field_pos_obj = field.getObject(name + "vision_pose")
+        self.best_log = field.getObject(name + "_best_log")
+        self.alt_log = field.getObject(name + "_alt_log")
+        self.field_pos_obj = field.getObject(name + "_vision_pose")
         self.pose_log_entry = wpiutil.log.FloatArrayLogEntry(
-            data_log, name + "vision_pose"
+            data_log, name + "_vision_pose"
         )
 
         self.chassis = chassis
         self.current_reproj = 0.0
+        self.has_multitag = False
 
     @feedback
     def reproj(self) -> float:
         return self.current_reproj
 
-    def execute(self) -> None:
-        # stop warnings in simulation
-        if wpilib.RobotBase.isSimulation():
-            return
+    @feedback
+    def multitag(self) -> bool:
+        return self.has_multitag
```

Comment:
For performance (and also for clarity when reading our logs), don't use `@feedback` to send a write-only field to NetworkTables. Use a `tunable` instead.

Reply:
I will change it, but that has got to be the most arse-backwards naming scheme! Tunable for something you don't want to actually tune, because if you want feedback you shouldn't use the type called "feedback". Wow. :)

Reply:
Does that mean all the feedback decorators should be changed?

Reply:
Why is this write only and why shouldn't this be a feedback method? The execute method sets this on line 88 and line 118.

Reply:
I have renamed it to make it clear it is providing feedback